contrib/ohi-release/README.md

```diff
@@ -0,0 +1,63 @@
+# 🛠️ `ohi-release`
+
+This is a wrapper of all the steps needed to update the changelog and render a snippet to be ready to be used as a release message. This contribution also includes a script that allow to replicate what this action does locally.
+
+## Example Usage
+
+```yaml
+- uses: newrelic/release-toolkit/contrib/ohi-release@v1
+  id: release
+- name: Commit updated changelog
+  run: |
+    git add CHANGELOG.md
+    git commit -m "Update changelog with changes from ${{ steps.release.outputs.next-version }}"
+    git push -u origin main
+    gh release create ${{ steps.release.outputs.release-title }} --target $(git rev-parse HEAD) --notes-file CHANGELOG.partial.md
+```
+
+## Parameters
+
+All parameters are optional:
+  * `excluded-dirs` exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".
+  * `fail-if-held` fails if the held toggle is active
```

Comment:
```suggestion
  * `fail-if-held` fails if the held toggle is active. Defaults to `true`.
```

Reply:
Fixed in [#2e447e6](https://github.com/newrelic/release-toolkit/pull/135/commits/2e447e6f41457d446032646f59654b6ce83c963f)